en/docs/reference/mediators/about-mediators.md

```diff
@@ -56,7 +56,7 @@ WSO2 Micro Integrator includes a comprehensive library of mediators that provide
 
 **Advanced Mediators**
 
-[Cache]({{base_path}}/reference/mediators/cache-Mediator) | [ForEach]({{base_path}}/reference/mediators/forEach-Mediator) | [Clone]({{base_path}}/reference/mediators/clone-Mediator) | [Store]({{base_path}}/reference/mediators/store-Mediator) | [Iterate]({{base_path}}/reference/mediators/iterate-Mediator) | [Aggregate]({{base_path}}/reference/mediators/aggregate-Mediator) | [Callout]({{base_path}}/reference/mediators/callout-Mediator) | [Transaction]({{base_path}}/reference/mediators/transaction-Mediator) | [Throttle]({{base_path}}/reference/mediators/throttle-Mediator) | [DBReport]({{base_path}}/reference/mediators/dB-Report-Mediator) | [DBLookup]({{base_path}}/reference/mediators/dBLookup-Mediator) | [EJB]({{base_path}}/reference/mediators/eJB-Mediator) | [Rule]({{base_path}}/reference/mediators/rule-Mediator) | [Binder]({{base_path}}/reference/mediators/builder-Mediator) | [Entitlement]({{base_path}}/reference/mediators/call-Mediator) | [OAuth]({{base_path}}/reference/mediators/call-Mediator) | [Smooks]({{base_path}}/reference/mediators/smooks-Mediator) | [Data Mapper]({{base_path}}/reference/mediators/data-Mapper-Mediator) | 
+[Cache]({{base_path}}/reference/mediators/cache-Mediator) | [ForEach]({{base_path}}/reference/mediators/forEach-Mediator) | [Clone]({{base_path}}/reference/mediators/clone-Mediator) | [Store]({{base_path}}/reference/mediators/store-Mediator) | [Iterate]({{base_path}}/reference/mediators/iterate-Mediator) | [Aggregate]({{base_path}}/reference/mediators/aggregate-Mediator) | [Callout]({{base_path}}/reference/mediators/callout-Mediator) | [Transaction]({{base_path}}/reference/mediators/transaction-Mediator) | [Throttle]({{base_path}}/reference/mediators/throttle-Mediator) | [DBReport]({{base_path}}/reference/mediators/dB-Report-Mediator) | [DBLookup]({{base_path}}/reference/mediators/dBLookup-Mediator) | [EJB]({{base_path}}/reference/mediators/eJB-Mediator) | [Binder]({{base_path}}/reference/mediators/builder-Mediator) | [Entitlement]({{base_path}}/reference/mediators/call-Mediator) | [OAuth]({{base_path}}/reference/mediators/call-Mediator) | [Smooks]({{base_path}}/reference/mediators/smooks-Mediator) | [Data Mapper]({{base_path}}/reference/mediators/data-Mapper-Mediator) | 
```

Comment:
@nilminiwso2 ,all the links to these file names with upper case will break. Please make them lower case.